src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine ex = new Excavator();
+        Machine tr = new Truck();
+        Machine bl = new Bulldozer();
+        Machine[] ar = {ex, tr, bl};
+
+        for (Machine i : ar) {
+            i.doWork();
+            i.stopWork();
```

Comment:
Ensure that the messages within the 'doWork' and 'stopWork' methods are informative and indicate what type of Machine is working. This is not visible in the MainApp.java file, but it's important to check the implementation of these methods in their respective classes.